services/merak-topo/handler/handler.go

```diff
@@ -437,9 +440,9 @@ func QueryHostNode(k8client *kubernetes.Clientset, topo_id string) error {
 	return nil
 }
 
-func Testapi(k8client *kubernetes.Clientset, topo database.TopologyData) ([]*common_pb.InternalComputeInfo, error) {
-	var cnodes []*common_pb.InternalComputeInfo
-	var cnode *common_pb.InternalComputeInfo
+func Testapi(k8client *kubernetes.Clientset, topo database.TopologyData) ([]*pb_common.InternalComputeInfo, error) {
+	var cnodes []*pb_common.InternalComputeInfo
+	var cnode *pb_common.InternalComputeInfo
```

Comment:
@zzengcs Please remove this func if you don't need it.

Reply:
Resolved